### ISSUE TITLE
BBC: Blead Breaks PJS/Tie-Array-BoundedIndex-0.06.tar.gz

### ISSUE THREAD

**Issue by eserte** [Perl/perl5#19465]
t/04attrib.t of the distribution's test suite started to fail with perl 5.35.9:
```
Invalid ARRAY attribute: Bounded(upper => 5) at t/04attrib.t line 22.
# Looks like your test exited with 255 just after 1.
t/04attrib.t .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/2 subtests 
```

**Comment by andk** [comment]
Bisect leads to v5.35.8-105-gf6387cff9c ; if you retry to bisect, pay proper attention that the perl has to have Test::Exception installed, otherwise a SKIP will prevent to find the proper commit. The reason is that the test is skipped when Test::Exception is not installed.

**Comment by jkeenan** [comment]
> Bisect leads to v5.35.8-105-gf6387cff9c ; if you retry to bisect, pay proper attention that the perl has to have Test::Exception installed, otherwise a SKIP will prevent to find the proper commit. The reason is that the test is skipped when Test::Exception is not installed.

I tried to confirm this finding via the following bisection invocation:
```
perl Porting/bisect.pl \
--start=f6387cff9cb31db4cf18c8641917ea4639ac2b65^ \
--end=v5.35.9 \
--with-module=Test::Exception \
--module=Tie::Array::BoundedIndex
```
However, AFAICT, there was no attempt to install Test::Exception.  All tests were run and PASSed.
```
Running make test for PJS/Tie-Array-BoundedIndex-0.06.tar.gz
/tmp/id0OhnFFUG/bin/perl Build --makefile_env_macros 1 test
t/01load.t .... ok   
t/02tie.t ..... ok   
t/03use.t ..... ok     
t/04attrib.t .. ok   
t/05pod.t ..... ok   
All tests successful.
```
What am I missing?

**Comment by haarg** [comment]
This was fixed by 90d32986170a76903fc0328ae6ac9e0ec4af40fc

**Comment by jkeenan** [comment]
> This was fixed by [90d3298](https://github.com/Perl/perl5/commit/90d32986170a76903fc0328ae6ac9e0ec4af40fc)

Appears so.
```
[perlmonger: blead] $ ./bin/perl -v | head -2 | tail -1
This is perl 5, version 35, subversion 10 (v5.35.10 (v5.35.9-20-g64f0d0d960)) built for amd64-freebsd-thread-multi
[perlmonger: blead] $ ./bin/perl -Ilib -MTest::Exception -MTie::Array::BoundedIndex -E 'say q|hello world|;'
hello world
```
